src/Features/CSharp/Portable/ConvertToInterpolatedString/CSharpConvertConcatenationToInterpolatedStringRefactoringProvider.cs

```diff
@@ -21,7 +21,13 @@ protected override bool SupportsInterpolatedStringHandler(Compilation compilatio
         => compilation.GetTypeByMetadataName("System.Runtime.CompilerServices.DefaultInterpolatedStringHandler") != null;
 
     protected override string GetTextWithoutQuotes(string text, bool isVerbatim, bool isCharacterLiteral)
-        => isVerbatim
-            ? text.Substring("@'".Length, text.Length - "@''".Length)
-            : text.Substring("'".Length, text.Length - "''".Length);
+    {
+        var contents = isVerbatim
+            ? text["@'".Length..^1]
```

Comment:
>? text["@'".Length..^1] [](http://example.com/codeflow?start=12&length=23)

These didn't include the end single quote previously, but now do?

Reply:
no.  this is the same as before, just with clearer syntax.  it's saying "take off two characters from the front, and one from the back"

Reply:
Oops, my brain read that wrong.